Vienna/Info.plist

```diff
@@ -139,7 +139,7 @@
 	<key>OSAScriptingDefinition</key>
 	<string>Vienna.sdef</string>
 	<key>SUEnableAutomaticChecks</key>
-	<true/>
+	<false/>
```

Comment:
This will also affect deployment builds. I doubt it is intentional ?

Reply:
Nope, just for testing to find the source of the problem. Clearly, this isn’t it.